docs/workflows/lambda.md

```diff
@@ -12,7 +12,7 @@ AWS provides great getting started guide for building Lambdas in the [developer
 
 Cumulus currently supports the following environments for Cumulus Message Adapter enabled functions:
 
-* [Node.js 10.16.3](https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html)
+* [Node.js 10.x, 12.x](https://docs.aws.amazon.com/lambda/latest/dg/programming-model.html)
```

Comment:
Nit: While *technically* true we should probably note that support for 10.x is not promised going forward.     (There's probably an a brief conversation to be had here about direction)